javascript/utils.js

```diff
@@ -130,6 +130,18 @@ function handleErrors (response) {
   return response
 }
 
+function safeString (str) {
+  return str != null ? str : ''
```

Comment:
Should we explicitly cast in these helper functions?

```js
return str != null ? String(str) : ''
```

Reply:
Super smart.